src/pages/library/SearchAndFilterComponent.ts

```diff
@@ -483,13 +485,13 @@ export const SearchAndFilterComponent = <ListItem>({
             htmlFor: id,
             style: { margin: '0 1ch 0 1rem', whiteSpace: 'nowrap' }
           }, ['Sort by']),
-          h(Select, {
+          h(SortSelect, {
             id,
             isClearable: false,
             isSearchable: false,
             styles: { container: old => ({ ...old, flex: '0 0 content' }) },
             value: sort,
-            onChange: ({ value }) => setSort(value),
+            onChange: opt => setSort(opt!.value),
```

Comment:
react-select's types say that onChange may be called with null. That seemed reasonable to leave in at the type level, so I've changed code that assumed a non-null value in onChange to use non-null assertions.